README.md

```diff
@@ -175,7 +173,7 @@ show:
   icon: false
 title_adaptive_color: true
 hours_to_show: 72
-name: 'https://www.google.com/'
+name: 'Google.com'
```

Comment:
We can't since the image print `https://www.google.com/`

Reply:
Reversed this change.